File: app/src/main/java/in/woloo/www/base/BaseEpoxyHolder.java
package in.woloo.www.base;

import android.view.View;

import androidx.annotation.CallSuper;

import com.airbnb.epoxy.EpoxyHolder;

import butterknife.ButterKnife;

public abstract class BaseEpoxyHolder extends EpoxyHolder {
    @CallSuper
    @Override
    protected void bindView(View itemView) {
        ButterKnife.bind(this, itemView);
    }
}

File: app/src/main/java/in/woloo/www/application_kotlin/base/BaseEpoxyHolder.kt
package `in`.woloo.www.application_kotlin.base

import android.view.View
import androidx.annotation.CallSuper
import butterknife.ButterKnife
import com.airbnb.epoxy.EpoxyHolder

abstract class BaseEpoxyHolder : EpoxyHolder() {
    @CallSuper
    override fun bindView(itemView: View) {
        ButterKnife.bind(this, itemView)
    }
}